Firestore/Example/FuzzTests/FSTFuzzTestsPrincipal.mm

```diff
@@ -18,14 +18,13 @@
 
 #include "LibFuzzer/FuzzerDefs.h"
 
-#include "Firestore/core/src/firebase/firestore/model/database_id.h"
-#include "Firestore/core/src/firebase/firestore/remote/serializer.h"
+#include "Firestore/Example/FuzzTests/FuzzingTargets/FSTFuzzTestSerializer.h"
+
 #include "Firestore/core/src/firebase/firestore/util/log.h"
 #include "Firestore/core/src/firebase/firestore/util/string_apple.h"
 
-using firebase::firestore::model::DatabaseId;
-using firebase::firestore::remote::Serializer;
 using firebase::firestore::util::MakeString;
+namespace fuzzing = firebase::firestore::fuzzing;
```

Comment:
Check out http://go/totw/119#scope-of-the-alias

We haven't quite been following all the advise to the letter in that link (in particular, we haven't been prefixing the global qualifier :: onto all of our user declarations), but one thing I'd like to do (where practical) is to move statements like this into the namespace scope. So in this case, move line 27 down below line 29.

It's a minor point, but should protect us if some other header dumps a symbol named 'fuzzing' into the global scope.

(This also applies for line 26.)

This assumes you *can* move them in without any undesirable effects. For instance, if these need to be defined outside the anon namespace to work with the rest of the code in the file, then don't bother. (We've got plenty of code like this; if we decide it's worth it, we'll clean it up all at once.)

Reply:
Thank for you the detailed feedback. I will move them inside the namespace. I went over the article to understand what's going on. It is pretty interesting and now I understand more about C++ namespaces.